beacon_node/beacon_chain/src/builder.rs

```diff
@@ -409,6 +409,11 @@ where
                 .init_blob_info(genesis.beacon_block.slot())
                 .map_err(|e| format!("Failed to initialize genesis blob info: {:?}", e))?,
         );
+        self.pending_io_batch.push(
```

Comment:
Could do this only if peer DAS is scheduled. Gives us a bit more freedom to change the on-disk format before the fork.